app/views/shared/_navigation_bar.html.slim

```diff
@@ -0,0 +1,6 @@
+nav.navbar.navbar-default
+  ul.nav.navbar-nav.navbar-center
+    li = link_to "Home", root_path
```

Comment:
@rubytaiwan/rubyists Which style do you prefer

`li=` ... (1)

or

`li =` ... (2)

?